### DISCUSSION TITLE
Authorization Pop-Up does not show on VS 2022 v17.2.3

### DISCUSSION THREAD

**Discussion by zakydzul** [community/community#17591] [Copilot Conversations]
The pop-up that should appear after restarting VS 2022 and creating a new project (as it instructed in getting started with VS) does not show up. is there any solutions to authorize the github copilot using another way?

**Comment by RobotX5** [comment]
i am having same issue 
It is not showing device info too.

**Comment by epsmae** [comment]
I am having same issue in 17.6.0 preview 4
GitHub Copilot 1.94.0.0 (v1.94.0.0@6586dd2df)
GitHub Copilot Agent v1.94.0
```
[DEBUG] [agent] [2023-07-24T08:35:15.123Z] Agent service starting
[DEBUG] [agent] [2023-07-24T08:35:15.556Z] Telemetry initialized
[INFO] [auth] [2023-07-24T08:35:16.129Z] Invalid copilot token: missing token: 404 
Auth Status: NotAuthorized
[INFO] [auth] [2023-07-24T08:35:16.486Z] Invalid copilot token: missing token: 404 
Auth Status: NotSignedIn
[INFO] [auth] [2023-07-24T08:35:16.918Z] Invalid copilot token: missing token: 404 
[INFO] [auth] [2023-07-24T08:35:17.101Z] Invalid copilot token: missing token: 404 
Auth Status: NotAuthorized
You don't have access to GitHub Copilot.
[INFO] Attempted sign in activeAccount https://github.com/22543513
```

**Comment by ranc** [comment]
I had the same issue,
Open the Account Settings dialog by going to File > Account Settings....
and refresh your github credentials.
just make sure you have an active license to co-pilot.

**Comment by ayim** [comment]
Going to close this issue since authentication has changed a lot since Visual Studio 17.2
